frontend/src/metabase/admin/permissions/components/PermissionsTable/PermissionsTable.jsx

```diff
@@ -102,10 +103,12 @@ export function PermissionsTable({
                 <PermissionsTableCell>
                   {entity.canSelect ? (
                     <EntityNameLink onClick={() => onSelect(entity)}>
-                      {entity.name}
+                      <Ellipsified>{entity.name}</Ellipsified>
                     </EntityNameLink>
                   ) : (
-                    <EntityName>{entity.name}</EntityName>
+                    <EntityName>
+                      <Ellipsified>{entity.name}</Ellipsified>
+                    </EntityName>
                   )}
 
                   {entity.hint && (
```

Comment:
this causes entity hints to get wrapped to the next line. I think they're meant to be on the same line

![Screenshot from 2022-04-20 16-30-05](https://user-images.githubusercontent.com/30528226/164335491-9eae3777-e501-4426-a811-4a083d23b5b0.png)